csharp/ql/src/semmle/code/csharp/frameworks/ServiceStack.qll

```diff
@@ -83,14 +83,19 @@ module Sources {
 
 /** Flow Sinks for the ServiceStack framework */
 module Sinks {
-  private import semmle.code.csharp.security.dataflow.flowsinks.Remote
+  private import semmle.code.csharp.security.dataflow.flowsinks.ExternalLocationSink
 
   /** RemoteFlow sinks for service stack */
-  class ServiceStackRemoteRequestParameter extends RemoteFlowSink {
+  class ServiceStackRemoteRequestParameter extends ExternalLocationSink {
```

Comment:
I guess `ExternalLocationSink` fits the bill. Is there any specific check and project on which you expect some changes due to this new sink?

Reply:
Not a specific check.  A customer was reporting zero findings using the service stack feature branch.  When we ran the debug query to find potential missing sinks, parameters to the IRestClient.Get() method came up.  So once the sink gets modeled correctly, there may be some checks that return results - I just don't know yet.